src/plugins/withDateProps.ts

```diff
@@ -1,19 +1,25 @@
 import { getDate } from 'date-fns'
 
 import { DateCell } from '../models'
-import { isSameDate, isSameYearAndMonth } from '../utils'
+import {
+  isSameDate,
+  isSameYearAndMonth,
+  isWeekend as _isWeekend,
+} from '../utils'
 
 export default function withDateProps(baseDate: Date, cursorDate: Date) {
   return function <T extends DateCell>(cell: T) {
     const { value: targetDate } = cell
     const isCurrentMonth = isSameYearAndMonth(cursorDate, targetDate)
     const isCurrentDate = isSameDate(baseDate, targetDate)
+    const isWeekend = _isWeekend(targetDate)
```

Comment:
just use `isWeekend` function in date-fns !

from. https://github.com/veccu/react-calendar/pull/38/files#r633473934